static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx

```diff
@@ -28,21 +28,12 @@ export function getWebVitalScores(data?: TableDataRow): ProjectScore {
     };
   }
   const scores = {
-    lcpScore: Math.round(
-      getWebVitalScore(data, 'lcp') * getWebVitalScoreWeight(data, 'lcp') * 100
-    ),
-    fcpScore: Math.round(
-      getWebVitalScore(data, 'fcp') * getWebVitalScoreWeight(data, 'fcp') * 100
-    ),
-    clsScore: Math.round(
-      getWebVitalScore(data, 'cls') * getWebVitalScoreWeight(data, 'cls') * 100
-    ),
-    ttfbScore: Math.round(
-      getWebVitalScore(data, 'ttfb') * getWebVitalScoreWeight(data, 'ttfb') * 100
-    ),
-    fidScore: Math.round(
-      getWebVitalScore(data, 'fid') * getWebVitalScoreWeight(data, 'fid') * 100
-    ),
+    lcpScore: Math.round(getWebVitalScore(data, 'lcp') * 100),
+    fcpScore: Math.round(getWebVitalScore(data, 'fcp') * 100),
+    clsScore: Math.round(getWebVitalScore(data, 'cls') * 100),
+    ttfbScore: Math.round(getWebVitalScore(data, 'ttfb') * 100),
+    fidScore: Math.round(getWebVitalScore(data, 'fid') * 100),
+    totalScore: Math.round(getTotalScore(data) * 100),
```

Comment:
These don't have to use weights?

Reply:
This function is supposed to get you the individual 0 - 100 webvital scores (used to display in the tile components) and total score, so we don't need to apply weights here